applications/crossbar/src/api_util.erl

```diff
@@ -42,6 +42,8 @@
 -define(MAX_UPLOAD_SIZE, whapps_config:get_integer(?CONFIG_CAT, <<"max_upload_size">>, 8000000)).
 
 -type halt_return() :: {'halt', cowboy_req:req(), cb_context:context()}.
+-type resource_exists() :: {boolean(), cb_context:context()}.
```

Comment:
I don't like this type's name :)

I don't have anything better than `resource_exists_return()` at the moment though. Maybe `resource_existence()`, since that's what you're conceptually returning?